.appveyor.yml

```diff
@@ -60,7 +60,7 @@ for:
 
     # setup go
     - rmdir c:\go /s /q
-    - "choco install golang"
+    - "choco install golang --version 1.15.7"
```

Comment:
Pinning down golang version to 1.15 as 1.16 changes default Windows installation directory which breaks AppVeyor.